### PR TITLE
Dropzone tidyup

### PR DESCRIPTION
**Pull request by axemonkey**
[Pivotal](https://www.pivotaltracker.com/story/show/138827415)

* Fix appearance of upload progress bar
* Style error messages as per sketch in Pivotal
* Ensure dropzone box resizes to fit multiple errors
* Add PDF to accepted file types in copy
* i18n

### CODE REVIEW

File: app/assets/javascripts/modules/doc-upload.js [hunk 1]
```diff
@@ -35,18 +35,29 @@ moj.Modules.docUpload = {
...
-      addRemoveLinks: true,
+      addRemoveLinks: false,
       createImageThumbnails: false,
       previewTemplate: previewTemplate,
-      dictRemoveFile: 'Remove',
       uploadMultiple: false,
       forceFallback: false,
       headers: {'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content')},
 
+      dictInvalidFileType: ' is in a format we don\'t accept. You can upload JPG, PNG, GIF, Word or Excel files.',
```
Comment (digitalronin) [inline app/assets/javascripts/modules/doc-upload.js]: PDFs, too. 

We have a list of allowed filetypes in the translation file. Can we reuse it here?

          <li>File formats: Word, PDF, Excel, JPG, GIF, PNG</li>

File: app/assets/javascripts/modules/doc-upload.js [hunk 2]
```diff
@@ -35,18 +35,29 @@ moj.Modules.docUpload = {
...
+      dictFileTooBig: ' is too big. You will need to resubmit a smaller version, less than ' + parseInt(self.$form.data('max-filesize')) + ' megabytes (MB).',
```
Comment (digitalronin) [inline app/assets/javascripts/modules/doc-upload.js]: If we can reuse the filetype list, can we also get this from the translation file? If it's a PITA, no worries - it's not a big deal

Comment (axemonkey) [inline app/assets/javascripts/modules/doc-upload.js]: I was definitely going to i18n these, but copy-wise I'm just going off what was in @mrbootle-moj 's designs. The sentences run on from the filename, so not sure we can re-use anything existing.

Comment (digitalronin) [inline app/assets/javascripts/modules/doc-upload.js]: Fair enough. In that case, please add PDF to the list, and let's just go with it as is.

Comment (axemonkey) [inline app/assets/javascripts/modules/doc-upload.js]: 👍

**Pull request merged**